activerecord/test/cases/validations_test.rb

```diff
@@ -187,9 +187,9 @@ def self.model_name
       validates_numericality_of :wibble, greater_than_or_equal_to: BigDecimal("97.18")
     end
 
-    assert_not_predicate klass.new(wibble: "97.179"), :valid?
-    assert_not_predicate klass.new(wibble: 97.179), :valid?
-    assert_not_predicate klass.new(wibble: BigDecimal("97.179")), :valid?
```

Comment:
These `assert_not_predicate` were originally odd/incorrect since the typecasted value `97.18` is valid for `greater_than_or_equal_to: BigDecimal("97.18")`.

Can you expand these to test both rounding up and down like https://github.com/rails/rails/pull/41019#discussion_r556250434?